dts/bindings/net/wireless/nordic,nrf-radio.yaml

```diff
@@ -29,94 +92,91 @@ properties:
     interrupts:
       required: true
 
+    dfe-supported:
```

Comment:
This binding should also include `802154-supported` or something like this, because only some radios support this protocol, namely nRF52811, nRF52820, nRF52833, nRF52840, nRF5340.